### PR TITLE
Metric date

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Enhancing examples https://dbt-labs.slack.com/archives/C02NCQ9483C/p1730927629130439

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-metric-date-dbt-labs.vercel.app/docs/build/measures

### CODE REVIEW

File: website/docs/docs/build/measures.md [hunk 1]
```diff
@@ -200,16 +200,16 @@ Parameters under the `non_additive_dimension` will specify dimensions that the m
 
 ```yaml
 semantic_models:
-  - name: subscription_id
+  - name: subscription
```
Comment (courtneyholcomb) [inline website/docs/docs/build/measures.md]: Can we make this `subscriptions` (plural)?

File: website/docs/docs/build/measures.md [hunk 2]
```diff
@@ -200,16 +200,16 @@ Parameters under the `non_additive_dimension` will specify dimensions that the m
...
     description: A subscription table with one row per date for each active user and their subscription plans. 
-    model: ref('your_schema.subscription_table')
+    model: ref('your_schema.subscription')
```
Comment (courtneyholcomb) [inline website/docs/docs/build/measures.md]: You can leave this how it was! The table name doesn't need to match the semantic model name.

File: website/docs/docs/build/measures.md [hunk 3]
```diff
@@ -200,16 +200,16 @@ Parameters under the `non_additive_dimension` will specify dimensions that the m
...
     defaults:
       agg_time_dimension: subscription_date
 
     entities:
       - name: user_id
         type: foreign
-        primary_entity: subscription_table
+    primary_entity: subscription
```
Comment (courtneyholcomb) [inline website/docs/docs/build/measures.md:212]: 👍🏼

Comment (courtneyholcomb) [reply website/docs/docs/build/measures.md:212]: This and everything below looks good!

**Review by courtneyholcomb — APPROVED** [review]

File: website/docs/docs/build/measures.md (hunk 1 above)
Comment (matthewshaver) [inline website/docs/docs/build/measures.md]:
```suggestion
  - name: subscriptions
```

Comment (matthewshaver) [reply website/docs/docs/build/measures.md]: Absolutely

File: website/docs/docs/build/measures.md (hunk 2 above)
Comment (matthewshaver) [inline website/docs/docs/build/measures.md]:
```suggestion
    model: ref('your_schema.subscription_table')
```

**Pull request merged**